rpc/opencdm_xdr.h

```diff
@@ -143,6 +143,10 @@ extern  rpc_response_generic * rpc_open_cdm_mediakeysession_release_1_svc(rpc_re
 extern  rpc_response_generic * rpc_open_cdm_mediaengine_1(rpc_request_mediaengine_data *, CLIENT *);
 extern  rpc_response_generic * rpc_open_cdm_mediaengine_1_svc(rpc_request_mediaengine_data *, struct svc_req *);
 extern int open_cdm_1_freeresult (SVCXPRT *, xdrproc_t, caddr_t);
+//WPE
+#define RPC_OPEN_CDM_MEDIAKEYS_SET_SERVER_CERTIFICATE 8
+#define RPC_OPEN_CDM_MEDIAKEYSESSION_REMOVE 9
+#define RPC_OPEN_CDM_MEDIAKEYSESSION_CLOSE 10
```

Comment:
>>Are these new defines used by anything currently?
No, it's not used by anything.

>>I guess this is the new RPC interface changes to match what was added in OpenCDM for the >>new EME APIs?
Yes.

>>It looks like we also need a minimal implementation for these, or at least a stub to print "not >>implemented" or similar.
>>With this commit it would also be good to get a more verbose commit message, why is it >>required?
default case handles that with some message.